### PR TITLE
#160451470 user can opt in and out notification

### PR DESCRIPTION
**Pull request by onwuvic**
#### What does this PR do?
This pull request adds functionality for user to opt-in and opt-out of notification

#### Description of Task to be completed?
- Create settings Component
- Create settings Reducers
- Create settings Actions

#### How should this be manually tested?
None

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#160451470

#### Questions:```
None

### CODE REVIEW

File: src/constants/Constants.js [hunk 1]
```diff
@@ -1,46 +1,55 @@
+const BASE_URL = 'https://phlox-ah-staging.herokuapp.com/api/v1';
 
-const BASE_URL = 'https://phlox-ah-staging.herokuapp.com';
 export const articleConstant = {
-  ALL_ARTICLES_URL: `${BASE_URL}/api/v1/articles/feed`,
-  MY_ARTICLES_URL: `${BASE_URL}/api/v1/myarticles`,
```
Comment (onwuvic) [inline src/constants/Constants.js:6]: Move the `api/v1/` to the base URL.

**Comment by oladimillion** [comment]
LGTM

File: src/containers/Header/Header.js [hunk 2]
```diff
@@ -301,6 +301,13 @@ class Header extends Component {
                   showSettingsOption
                     && <div className="sd-wrapper">
                       <ul>
+                      <li
+                        className="s-list">
+                        <Link to="/settings">
+                          <i className="fas fa-cog"></i>
+                          &nbsp; Settings
```
Comment (s-akinrele) [inline src/containers/Header/Header.js]: can you use css.. give it a margin

File: src/containers/Settings/Settings.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+  /**
+   * @description - This method sets the input values
+   * @param {object} checked
+   * @param {object} event
+   * @param {object} id
+   * @returns {object} void
+   * @memberof Settings
+   */
+  handleChange(checked, event, id) {
```
Comment (s-akinrele) [inline src/containers/Settings/Settings.js]: toggleButton

File: src/containers/Settings/Settings.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+class Settings extends Component {
+  /**
+   * @description - This method runs first in the class
+   * @returns {object} Settings
+   * @memberof Settings
+   */
+  constructor() {
+    super();
+
+    this.state = { notificationStatus: false };
```
Comment (s-akinrele) [inline src/containers/Settings/Settings.js]: define check in state.....

File: src/containers/Settings/Settings.js (hunk 3 above)
Comment (s-akinrele) [reply src/containers/Settings/Settings.js]: refactor as discussed

**Review by s-akinrele — CHANGES_REQUESTED** [review]
I had some comments

File: src/index.js [hunk 5]
```diff
@@ -18,10 +18,6 @@ import './styles/style.scss';
 import App from './App';
 import rootReducer from './reducers/Index';
 
-if (localStorage.token) {
-  setAuthToken(localStorage.token);
```
Comment (onwuvic) [inline src/index.js:22]: This code was removed because it is a duplication of an existing code

**Review by s-akinrele — APPROVED** [review]

**Pull request merged**